.github/actions/simple-md-toc/action.yml

```diff
@@ -0,0 +1,57 @@
+#Github Composite actions to generate Markdown Table of Contents
+name: Simple Markdown ToC Generator
+description: Generate a Table of Contents from a Markdown file
+inputs:
+  md_toc_version:
+    description: Markdown Table of Contents Package version
+    required: false
+    default: 1.2.0
+  md_src:
+    description: Markdown file to generate Table of Contents
+    required: true
+  depth:
+    description: Depth of Table of Contents
+    required: false
+    default: '2'
+  dry_run:
+    description: Skip destructive actions
+    required: false
+    default: 'false'
```

Comment:
is this really needed? pushing a commit is not really destructive and the action would just change the files locally without doing any harm